database/loadVolumeData.js

```diff
@@ -148,15 +149,27 @@ function loadPolicies (startId, orgId, teamId, callback) {
   })
 }
 
+function getMetaData (val1, val2) {
+  if (ADD_METADATA) {
+    var obj = {
+      key1: val1,
+      key2: val2
+    }
+    return "'" + JSON.stringify(obj) + "'::JSONB"
```

Comment:
Why does `::JSONB` need to be appended to the stringified json?

Reply:
Also swagger needs to be regenerated in order to update `/docs`, doesn't have to be done as part of this PR but we should review documentation before this gets published.

Reply:
I committed swagger docs in the latest commit :-)

Reply:
please make sure you release the client, although in this particular case it gives no side effects.

```
client.query('SELECT NOW()', (err, res) => {
  console.log(err, res)
  client.end()
})
```

Reply:
ok on termination of process?  I'm using the same client throughout.